### ISSUE TITLE
Filters not available in Query Editor 

### ISSUE THREAD

**Issue by joshughes** [alexanderzobnin/grafana-zabbix#293]
#### I'm submitting a ...
 Question / Support request

#### For bug report please include this information:
- Grafana 3.1.1-1470047149
- Zabbix 3.0
- zabbix plugin is latest version
- Ubuntu 14.04

I am not sure why I am unable to see the filters while building my graphs. Its not clear if it is because of the version of Zabbix I am using or if there might be something misconfigured somewhere. 

![Uploading Screen Shot 2016-10-12 at 9.01.36 AM.png…]()

**Comment by alexanderzobnin** [comment]
@joshughes can you reload screenshots?

**Comment by joshughes** [comment]
THis was just a documentation issue... The docs show a filter box that I assume is for an older version. New version you just have to add the regex right into the field.

**Comment by alexanderzobnin** [comment]
@joshughes  Where are you looking for the docs? I added getting started guide: http://docs.grafana-zabbix.org/guides/gettingstarted/

**Comment by joshughes** [comment]
I ended up on this page... because I was trying to get some information about templates. 
https://github.com/alexanderzobnin/grafana-zabbix/wiki/Usage#templates

On that page there are a few outdated screen shots... When I found my way back to your getting started page... I realized that the screens there were up to date. 

Basically I have a need for templates and regex in data sources... so I was going back and forth between those two resources and got a bit confused.
